src/main/java/commandparser/StringParser.java

```diff
@@ -0,0 +1,131 @@
+package commandparser;
+
+import data.TaskManagerException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class StringParser {
```

Comment:
StringParser is a very useful class since we primarily deal with that throughout the whole app, so this makes it very reusable, good job!